doc/main.html

```diff
@@ -937,43 +937,57 @@ <h4>Internal code snippets</h4>
       <section id="sec-note-class">
         <h4>Notes</h4>
 
-        <p>Notes can be inserted by using <code>p</code> or <code>div</code> element whose <code>class</code> attribute is equal to
-        <code>note</code>.</p>
+        <p>A note is inserted by using a <code>p</code> or <code>div</code> element whose <code>class</code> attribute is equal to
+        <code>note</code>. The element element can also have an <code>id</code> attribute, in which case the value of the
+        <code>id</code> attribute shall start with the string <code>note-</code>.</p>
 
-        <p>Notes will automatically be formatted, e.g. prepended with the text <i>NOTE</i>, and numbered.</p>
+        <p>Notes will automatically be formatted and numbered.</p>
 
         <div class="example">
 <pre>
-&lt;p class=&quot;note&quot;&gt;
+&lt;p id="note-example-ref" class=&quot;note&quot;&gt;
   The license portion of Annex A is verbatim the BSD-3-Clause license
   available at &lt;a&gt;https://spdx.org/licenses/BSD-3-Clause.html&lt;/a&gt;.
 &lt;/p&gt;
 </pre>
 
           <p>is rendered as</p>
 
-          <p class="note">The license portion of Annex A is verbatim the BSD-3-Clause license available at
+          <p id="note-example-ref"  class="note">The license portion of Annex A is verbatim the BSD-3-Clause license available at
           <a>https://spdx.org/licenses/BSD-3-Clause.html</a>.</p>
         </div>
 
+        <p>A note can also be cited by referencing the value of the <code>id</code> attribute.</p>
+
+        <div class="example">
+          <code>&lt;a href=&quot;#note-example-ref&quot;&gt;&lt;/a&gt;</code> results in <a href="#note-example-ref"></a>.
+        </div>
+
       </section>
 
       <section id="sec-example-class">
         <h4>Examples</h4>
 
-        <p>Examples can be inserted by using <code>div</code> element whose <code>class</code> attribute is equal to
-        <code>example</code>.</p>
+        <p>An example is inserted by using <code>div</code> element whose <code>class</code> attribute is equal to
```

Comment:
```suggestion
        <p>An example is inserted by using a <code>p</code> or <code>div</code> element whose <code>class</code> attribute is equal to
```

Reply:
This still applies, yes?

Reply:
Hadn't we decided that examples could only be `div`?

Reply:
I though we expanded for `div`, not disallowed `p`. Both currently work. I'm fine with it either way tbh.

Reply:
Ok. Let me add tests for both.